config/tests/guest/libvirt/pci_passthrough.cfg

```diff
@@ -0,0 +1,55 @@
+include tests-shared.cfg
+username = root
+password = 123456
+main_vm = virt-tests-vm1
+vms = virt-tests-vm1
+#Network
+nettype = bridge
+netdst=virbr0
+# Using Text mode of installation
+display = 'nographic'
+take_regular_screendumps = no
+keep_screendumps_on_error = no
+keep_screendumps = no
+store_vm_register = no
+virt_install_binary = /usr/bin/virt-install
+qemu_img_binary = /usr/bin/qemu-img
+hvm_or_pv = hvm
+machine_type = pseries
+only bridge
+no xen, lxc, esx, ovmf
+#Filterout unwanted disk types
+no ide,xenblk,lsi_scsi,ahci,sd
+no qed,qcow2v3,raw_dd,vmdk, usb2
+no e1000-82540em,e1000-82545em,e1000-82544gc,xennet,nic_custom
+only no_virtio_rng
+only smp2
+only no_9p_export
+only no_pci_assignable
+only (image_backend=filesystem)
+only smallpages
+smp = 32
+vcpu_cores = 32
+vcpu_threads = 1
+vcpu_sockets = 1
+# 32G
+mem = 32768
+number_vfs = "Enter the no.of Virtual Function's to be created for each PF"
+# number_vfs = 2
```

Comment:
same here

Reply:
@sathnaga Addressed the changes, please review.